conans/client/rest/auth_manager.py

```diff
@@ -193,5 +193,5 @@ def authenticate(self, user, password):
             raise ConanException("Password contains not allowed symbols")
 
         # Store result in DB
-        update_localdb(self._localdb, user, token, self._remote, self._user_io.out)
-        return token
+        remote, prev_user, user = update_localdb(self._localdb, user, token, self._remote)
+        return token, remote, prev_user, user
```

Comment:
Who is using this returned objects? I can't see their usage.

Reply:
They are used in the command.py to send them to the output printer